src/Template/Vocabulary/add_sentences.ctp

```diff
@@ -55,6 +60,13 @@ $this->set('title_for_layout', $this->Pages->formatTitle($title));
             )
             ?>
         </div>
+        <div class="empty-info-text">
```

Comment:
When there is at least one request, this div will show up as empty, potentially creating a blank space. I think it would be better to make it *not show up at all* by putting it *inside* the `if()` branch.

Reply:
good point!